core/app/[locale]/(default)/page.tsx

```diff
@@ -68,7 +68,7 @@ export default async function Home({ params: { locale } }: Props) {
       <FeaturedProductsCarousel products={newestProducts} title={t('Carousel.title')} />
 
       <FeaturedImage
-        cta={{ href: '/#', label: t('FeaturedImage.cta') }}
+        cta={{ href: '/shop-all', label: t('FeaturedImage.cta') }}
```

Comment:
@jorgemoya, can you confirm if this is the way we want to add links? Do they need to be localized using `next-intl`?

Reply:
I did a quick scan. I searched through `href` references and I didn't see anything existing that included a localized URL as far as I could tell.